docs/docs/app/datasources.mdx

```diff
@@ -23,6 +23,7 @@ Below are the currently supported Data Sources (links are to our detail guides):
 - MySQL/MariaDB
 - MsSQL/SQL Server
 - PrestoDB (and Trino)
+- Databricks (Delta Lake)
```

Comment:
Is it `Delta Lake`? That seems like something Databricks leverages but is a separate concept altogether.

Not sure myself.

Reply:
I'm not sure either.  I can remove the Delta Lake part.